src/devices/philips.ts

```diff
@@ -2506,6 +2506,13 @@ const definitions: Definition[] = [
         description: 'Hue Lily outdoor spot light',
         extend: philips.extend.light_onoff_brightness_colortemp_color({colorTempRange: [153, 500]}),
     },
+    {
+        zigbeeModel: ['LCS002'],
+        model: '577262',
```

Comment:
When googling on this number, I don't get this bulb, either:

- Use `046677577261` here (EAN)
- Or a model number that is written on the device/box, I expect something in the format of `1742830P7`

Reply:
Ah thanks, I updated this to match what is on the device itself. In Googling 9290031508, I see a lot of indications that it should have an X at the end (e.g. FCC website), but the device does not have the X on the model number.